source/atomics.tex

```diff
@@ -1999,7 +2000,7 @@
 \end{note}
 \begin{note}
 For a union with bits that participate in the value representation
-of some members but not others, compare-and-exchange might always fail.
+of some members but not others, compare-and-exchange can always fail.
```

Comment:
Maybe "... but not others, it is possible for compare-and-exchange to always fail."  ?

Or "it is possible that a compare-and-exchange operation will never succeed."

Reply:
fixed and force-pushed